Formula/afflib.rb

```diff
@@ -57,21 +65,24 @@ def install
  aclocal
  LIBTOOLIZE=glibtoolize
 diff --git a/configure.ac b/configure.ac
-index 940353b..c530f2e 100644
+index 7de2900..5c1ce82 100644
 --- a/configure.ac
 +++ b/configure.ac
-@@ -241,10 +241,6 @@ AC_ARG_ENABLE(fuse,
+@@ -238,13 +238,6 @@ AC_ARG_ENABLE(fuse,
  if test "x${enable_fuse}" = "xyes" ; then
    AC_MSG_NOTICE([FUSE requested])
    CPPFLAGS="-D_FILE_OFFSET_BITS=64 -DFUSE_USE_VERSION=26 $CPPFLAGS"
 -  if test `uname -s` = Darwin ; then
 -    AC_MSG_NOTICE([FUSE IS NOT SUPPORTED ON MACOS])
 -    enable_fuse=no
 -  fi
```

Comment:
Great. Can we use https://github.com/sshock/AFFLIBv3/pull/13.patch as a patch URL instead?